pkg/token/token.go

```diff
@@ -52,6 +52,9 @@ type Identity struct {
 	// UserID is the unique user/role ID (e.g., "AROAAAAAAAAAAAAAAAAAA").
 	UserID string
 
+	// IAM user or role name
```

Comment:
If we end up keeping this Go comments start with the name of the thing being commented, in this case `Name` and are complete sentences.